src/main/cpp/VideoState.h

```diff
@@ -72,6 +72,11 @@ extern "C" {
 // fixed point to double
 #define CONV_FP(x) ((double) (x)) / (1 << 16)
 
+#define ENABLE_SHOW_STATUS 1
```

Comment:
If this macro is defining an internal status used only in the VideoState Class. I would prefer using constant data members over defining Macros in order to control the scope of members.

Reply:
It's used in three places--but I've pushed it as internal constant into all three classes.